<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CirclesDialog</class>
 <widget class="QWidget" name="CirclesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="CirclesDialogVLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="titleBarFrame">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QHBoxLayout" name="titleBarFrameHLayout">
      <property name="margin">
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="titleBarPixmap">
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../images.qrc">:/images/circles/circles_32.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="StyledLabel" name="titleBarLabel">
        <property name="text">
         <string>Circles</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_localCircle">
        <property name="text">
         <string>Create Personal Circle</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_extCircle">
        <property name="text">
         <string>Create External Circle</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_editCircle">
        <property name="text">
         <string>Edit Circle</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="todoPushButton">
        <property name="text">
         <string>Todo</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="titleBarHSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>378</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="showingDetailsLabel">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Showing details: </string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="mainHLayout">
     <item>
      <widget class="QGroupBox" name="memberGBox">
       <property name="title">
        <string>Membership</string>
       </property>
       <layout class="QVBoxLayout" name="memberGBoxVLayout">
        <item>
         <widget class="QTreeWidget" name="treeWidget_membership">
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>IDs</string>
           </property>
          </column>
          <item>
           <property name="text">
            <string>Public Circles</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Personal Circles</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="peerGBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="title">
        <string>Peers</string>
       </property>
       <layout class="QVBoxLayout" name="peerGBoxVLayout">
        <item>
         <widget class="QTreeWidget" name="treeWidget_friends">
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Status</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>ID</string>
           </property>
          </column>
          <item>
           <property name="text">
            <string>Others</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Friends of Friends</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Friends</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="permissionsGBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Permissions</string>
       </property>
       <layout class="QVBoxLayout" name="permissionsGBoxVLayout">
        <item>
         <widget class="QCheckBox" name="checkBox_anon">
          <property name="text">
           <string>Anon Transfers</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_disc">
          <property name="text">
           <string>Discovery</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeWidget" name="treeWidget_category">
          <column>
           <property name="text">
            <string>Share Category</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StyledLabel</class>
   <extends>QLabel</extends>
   <header>gui/common/StyledLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
